client/lib/desktop/index.js

```diff
@@ -117,7 +122,7 @@ var Desktop = {
 
 	onShowMySites: function() {
 		debug( 'Showing my sites' );
-		const site = this.getSelectedSite();
```

Comment:
@timmyc how did this originally work? If we do have access to selected site by different means, then this PR can be much smaller.